content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -534,18 +534,33 @@ please install corresponding CSI drivers.
 * [`vsphereVolume`](/docs/concepts/storage/volumes/#vspherevolume) - vSphere VMDK volume
   (**migration on by default** starting v1.25)
 
+The following PersistentVolume types are supported through the `CSIMigration` feature gate and require the corresponding CSI driver to be installed in the cluster. Kubernetes offers built-in APIs to facilitate their use.
```

Comment:
Is that right? AIUI there is no longer a `CSIMigration` feature gate in current Kubernetes.

Reply:
@sftim, this was worded as per the advice received from @msau42.
@msau42, request you to kindly review if these sentences are appropriate.

Reply:
I agree with @sftim, CSIMigration feature gate is now removed.

Reply:
This comment is not addressed yet.  The core CSIMigration feature is GA and the feature gate is removed.
Suggest change to:
The following PersistentVolume types support CSIMigration and require the corresponding CSI driver to be installed in the cluster. Kubernetes offers built-in APIs to facilitate their use.

Reply:
@xing-yang This has been incorporated in the latest commit.